connectd/connectd.c

```diff
@@ -358,7 +361,12 @@ struct io_plan *peer_connected(struct io_conn *conn,
 		 * in progress right now) */
 		if (connect->conn)
 			io_set_finish(connect->conn, NULL, NULL);
+		connect_reason = tal_steal(tmpctx, connect->reason);
+		connect_time_nsec = time_to_nsec(timemono_since(connect->start));
 		tal_free(connect);
+	} else {
+		connect_reason = "";
```

Comment:
Would it be helpful to explicitly label an "unrequested inbound connection" here?